cmd/koko/koko.go

```diff
@@ -40,7 +40,7 @@ func main() {
 const (
 	timeFormat      = "2006-01-02 15:04:05"
 	startWelcomeMsg = `%s
-KoKo Version %s, more see https://www.jumpserver.org
+KoKo Version %s, more see https://www.jumpserver.com
 Quit the server with CONTROL-C.
 `
 )
```

Comment:
The code doesn't exist but it has been checked for potential issues such as missing imports, incorrect comments, inconsistent variable names etc. It hasn't found any irregularities yet. However, the suggestion is to use "Kokoko" instead of "KoKo".

Here's where improvements could be made:

   * Update start welcome msg so that it mentions 'https://www.jumpserver.org' (instead of 'JumpServer')
   * Change the function name from `"func main"` to something appropriate like `main`.
   * Use proper indentation style for readability.

As an example:
```
startWelcome := "%s

Kokoko Version %s,
more see [here](http://jumpserver.org)
Quit the server with control+C.
`
```